### PR TITLE
[plugin.video.viervijfzes@matrix] 0.4.1+matrix.1

### PR DESCRIPTION
**Pull request by michaelarnauts**
### Add-on details:

- **General**
  - Add-on name: GoPlay
  - Add-on ID: plugin.video.viervijfzes
  - Version number: 0.4.1+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/add-ons/plugin.video.viervijfzes
  
This add-on gives access to video-on-demand content available on the websites of Play4, Play5 and Play6.

### Description of changes:

v0.4.1 (2021-02-27)
- Add integration with My List on GoPlay.
- Add categories and recommendations.
- Solves an issue with updating metadata in the background.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.viervijfzes/resources/lib/viervijfzes/aws/cognito_identity.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+""" Amazon Cognito Identity implementation without external dependencies """
```
Comment (enen92) [inline plugin.video.viervijfzes/resources/lib/viervijfzes/aws/cognito_identity.py:2]: What is this used for?

**Review by enen92 — COMMENTED** [review]
The usage of cognito/aws needs clarification. @basrieter @Lunatixz can you guys also take a look?

File: plugin.video.viervijfzes/resources/lib/viervijfzes/aws/cognito_identity.py (hunk 1 above)
Comment (michaelarnauts) [reply plugin.video.viervijfzes/resources/lib/viervijfzes/aws/cognito_identity.py:2]: Amazon Cognito is the authentication service used by GoPlay. You authenticate with amazon cognito, and you get a token back that you can use to talk to the API.

It's also used by Retrospect of @basrieter

**Review by enen92 — APPROVED** [review]

File: plugin.video.viervijfzes/resources/lib/viervijfzes/aws/cognito_identity.py (hunk 1 above)
Comment (basrieter) [reply plugin.video.viervijfzes/resources/lib/viervijfzes/aws/cognito_identity.py:2]: That is correct. They use AWS as identity provider.

**Review by basrieter — APPROVED** [review]

**Comment by michaelarnauts** [comment]
Thanks for your time, @enen92!

**Pull request merged**